hack/functest.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -ex
+
+function new_test() {
+    name=$1
+
+    printf "%0.s=" {1..80}
+    echo
+    echo ${name}
+}
+
+new_test 'Test e2e Node Mainenance'
```

Comment:
Please add `funktest.sh` and `sync.sh` in the same commit that removes the old `cluster` directory, so that `git` can correctly work out that the files are being moved.
Otherwise there is no way to follow the history of this file - as it appears to come from nowhere in this commit

Reply:
done.